src/main/java/com/meta/cp4m/store/NullStore.java

```diff
@@ -0,0 +1,34 @@
+/*
+ *
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.meta.cp4m.store;
+
+import com.google.common.cache.Cache;
+import com.meta.cp4m.Identifier;
+import com.meta.cp4m.message.Message;
+import com.meta.cp4m.message.ThreadState;
+
+import java.util.List;
+
+public class NullStore<T extends Message> implements ChatStore<T> {
```

Comment:
let's add a brief [javadoc](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html) here explaining that this is the default that doesn't store any data